tests/FakeItEasy.Specs/ConfiguringPropertySetterSpecs.cs

```diff
@@ -54,6 +55,7 @@ public interface IHaveInterestingProperties
                 .x(() => exception.Should().BeAnExceptionOfType<FakeConfigurationException>());
         }
 
+        [SuppressMessage("Microsoft.Globalization", "CA1305:SpecifyIFormatProvider", MessageId = "System.String.Format(System.String,System.Object)", Justification = "Not relevant in this case")]

```

Comment:
Alternatively, I could use `FormattableString.Invariant`. Since the project doesn't target .NET 4.6, it would require adding the [StringInterpolationBridge](https://www.nuget.org/packages/StringInterpolationBridge) package.